adapters/faraday/lib/opentelemetry/adapters/faraday/middlewares/tracer_middleware.rb

```diff
@@ -11,35 +11,40 @@ module Middlewares
         # TracerMiddleware propagates context and instruments Faraday requests
         # by way of its middlware system
         class TracerMiddleware < ::Faraday::Middleware
+          HTTP_METHODS_SYMBOL_TO_STRING = {
+            connect: 'CONNECT',
+            delete: 'DELETE',
+            get: 'GET',
+            head: 'HEAD',
+            options: 'OPTIONS',
+            patch: 'PATCH',
+            post: 'POST',
+            put: 'PUT',
+            trace: 'TRACE'
+          }.freeze
+
           def call(env)
-            return app.call(env) if disable_span_reporting?(env)
+            http_method = HTTP_METHODS_SYMBOL_TO_STRING[env.method]
 
-            tracer.in_span(env.url.to_s,
-                           attributes: { 'component' => 'http',
-                                         'http.method' => env.method,
-                                         'http.url' => env.url.to_s },
-                           kind: :client) do |span|
-              propagate_context(span, env)
+            tracer.in_span(
+              "HTTP #{http_method}",
+              attributes: {
+                'component' => 'http',
+                'http.method' => http_method,
+                'http.url' => env.url.to_s
+              },
+              kind: :client
+            ) do |span|
+              propagator.inject(span.context, env.request_headers)
 
               app.call(env).on_complete { |resp| trace_response(span, resp) }
             end
           end
 
-          # Override implementation (subclass) to determine per-connection
-          # span reporting rules.
-          def disable_span_reporting?(_env)
-            false
-          end
-
           private
 
           attr_reader :app
 
-          # Outbound requests should only need to inject the current span.
-          def propagate_context(span, env)
-            propagator.inject(span.context, env.request_headers)
-          end
-
           def propagator
             OpenTelemetry.tracer_factory.http_text_format
           end
```

Comment:
This can probably be removed, unless you'd prefer to delegate to `OpenTelemetry.propagation`.